### PR TITLE
Connect to open weather API.

### PR DESCRIPTION
**Pull request by marcianosr**

### CODE REVIEW

File: src/js/main.js [hunk 1]
```diff
@@ -804,6 +806,95 @@ swiper.addEventListener(
 	supportsPassive ? { passive: true } : false
 );
 
+const weatherSection = document.querySelector(".weather-section");
+
+async function getWeather() {
+	const response = await fetch(
+		"http://api.openweathermap.org/data/2.5/weather?q=Salto,uy?&units=metric&APPID=b0142355ff1172118bcf173d1bd9f022"
```
Comment (RoelN) [inline src/js/main.js]: Shouldn't this be Arapey?

Comment (RoelN) [reply src/js/main.js]: Is it possible to do a `https` request?

File: src/js/main.js [hunk 2]
```diff
@@ -804,6 +806,95 @@ swiper.addEventListener(
...
+			max: 50,
+			min: 0
+		},
+		humidity: {
+			current: humidity,
+			max: 100,
+			wind
+		},
+		time: {
+			current: "2:00"
```
Comment (RoelN) [inline src/js/main.js]: This seems to remain hardcoded?

File: src/js/main.js [hunk 3]
```diff
@@ -804,6 +806,95 @@ swiper.addEventListener(
...
+		};
+	},
+	time: () => {
+		const date = new Date();
+		const hours = date.getHours();
+		const minutes = date.getMinutes();
+
+		return {
+			weight: 400,
+			text: `Thursday 14:00`
```
Comment (RoelN) [inline src/js/main.js]: Can we get Arapey time here? :-)

File: src/js/main.js (hunk 1 above)
Comment (marcianosr) [reply src/js/main.js]: Well I tried to look for "Arapey" or "Termas del Arapey", but they can't be found. Salto is the nearest location that could be found though.

**Review by RoelN — APPROVED** [review]

**Pull request merged**